pkgs/by-name/ma/mattermost/update.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq common-updater-scripts prefetch-npm-deps
+set -euo pipefail
+
+version_regex='^v9\.11\.[0-9]+$'
+
+# Get latest ESR version (9.11.x series)
```

Comment:
Possibly move this comment before the version regex. I think in the future we may use two version regexes (ESR and latest).